docs/one-click-integrations/bitbucket-integration.md

```diff
@@ -47,6 +59,23 @@ If there is an error please [contact support](mailto:pulsesupport@codacy.com).
 -   The set of changes in a deployment is the list of commits in the corresponding pull request.
 -   Pulse associates all the Bitbucket user groups of the author of a merged pull request with the corresponding deployment, excluding the user groups with less than two members. Pulse only takes the changes to Bitbucket user groups into account on pull requests merged after those changes.
 
+## Automatic incident detection strategies {: id="incident-detection-strategy"}
+
+The following is a detailed description of how the Pulse Bitbucket integration automatically detects incidents:
```

Comment:
I'd expand this into a list of links. See this [related comment](https://github.com/codacy/pulse-user-docs/pull/195/files#r1108187851) on the other PR.

Reply:
Thanks for the feedback, @nicklem !
I rephrased the introductory sentence for clarity, linking fort the sections below. In this case, I believe a list of strategies wouldn't apply, as we only have one strategy. The second option is to not detect incidents.